lib/caps/ICL/iHD

```diff
@@ -64,6 +64,7 @@ caps = dict(
     denoise     = dict(
       ifmts = ["NV12", "P010", "YUY2"],
       ofmts = ["NV12", "P010", "YUY2"],
+      effect = ["Luma"],
```

Comment:
Can we use `chroma = False` instead?  Then use `self.caps.get("chroma", True)` to check for the cap.  This would follow the other style better (e.g. `multislice` in vdenc avc lib/caps/KBL/i965).